src/test/java/Search.java

```diff
@@ -0,0 +1,59 @@
+import InternSearch.pages.LoginPageSearch;
+import InternSearch.pages.SearchPageSearch;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Search extends BaseTest {
```

Comment:
**According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. 
2. Use whole words and must avoid acronyms and abbreviations.

This class should be renamed to something more descriptive , like `public class SearchTests`

Reply:
I renamed it as your suggestion. I learned a lot from your feedback. Thank you.